Sources/SwiftParser/Declarations.swift

```diff
@@ -60,7 +60,9 @@ extension TokenConsumer {
         modifierProgress.evaluate(subparser.currentToken)
       {
         subparser.eat(handle)
-        if subparser.at(.leftParen) && modifierKind.canHaveParenthesizedArgument {
+        if modifierKind != .open && subparser.at(.leftParen) && modifierKind.canHaveParenthesizedArgument {
```

Comment:
Why only for `open`?

Reply:
Because all the other access modifiers are lexer classified keywords and thus can’t be used as function base names.